ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/fastsync/worldstate/FastWorldDownloadState.java

```diff
@@ -71,6 +74,10 @@ public synchronized boolean checkCompletion(final BlockHeader header) {
       // so they can give up waiting.
       notifyAll();
       LOG.info("Finished downloading world state from peers");
+
+      // stop the metrics timer for the world download
```

Comment:
I feel like this comment could be incorporated into either the log message above or the method call below

Reply:
this has been removed